taxonomy-editor-frontend/src/pages/editentry/ListTranslations.tsx

```diff
@@ -95,8 +96,10 @@ export const ListTranslations = ({
 
   const shownLanguagesInfo = languagesToShow.map((languageCode: string) => {
     const languageName =
-      ISO6391.getName(languageCode) +
-      (languageCode === nodeObject.main_language ? " (main language)" : "");
+      languageCode === "xx"
+        ? "All languages"
+        : ISO6391.getName(languageCode) +
```

Comment:
```suggestion
      (languageCode === "xx"
        ? "All languages"
        : ISO6391.getName(languageCode)) +
```

I think the ternary should be between parentheses, so we display "All languages (main language)" if xx is the main language

Reply:
I think this comment is still relevant 😅 (although I'm not sure xx can be a main language)